hack/install-build.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Any particular reason to have a separate script rather than a make target for this?

Reply:
this was in the makefile at one point, but it was very system / user preference dependent. This script is to avoid forcing anyone to develop in a particular way